recipe/meta.yaml

```diff
@@ -118,141 +77,177 @@ outputs:
+            {{ opencv_libs.append("imgproc") or "" }}
+            {{ opencv_libs.append("intensity_transform") or "" }}
+            {{ opencv_libs.append("line_descriptor") or "" }}
+            {{ opencv_libs.append("mcc") or "" }}
+            {{ opencv_libs.append("ml") or "" }}
+            {{ opencv_libs.append("objdetect") or "" }}
+            {{ opencv_libs.append("optflow") or "" }}
+            {{ opencv_libs.append("phase_unwrapping") or "" }}
+            {{ opencv_libs.append("photo") or "" }}
+            {{ opencv_libs.append("plot") or "" }}
+            {{ opencv_libs.append("quality") or "" }}
+            {{ opencv_libs.append("rapid") or "" }}
+            {{ opencv_libs.append("reg") or "" }}
+            {{ opencv_libs.append("rgbd") or "" }}
+            {{ opencv_libs.append("saliency") or "" }}
+            {{ opencv_libs.append("shape") or "" }}
+            {{ opencv_libs.append("stereo") or "" }}
+            {{ opencv_libs.append("stitching") or "" }}
+            {{ opencv_libs.append("structured_light") or "" }}
+            {{ opencv_libs.append("superres") or "" }}
+            {{ opencv_libs.append("surface_matching") or "" }}
+            {{ opencv_libs.append("text") or "" }}
+            {{ opencv_libs.append("tracking") or "" }}
+            {{ opencv_libs.append("video") or "" }}
+            {{ opencv_libs.append("videoio") or "" }}
+            {{ opencv_libs.append("videostab") or "" }}
+            {{ opencv_libs.append("wechat_qrcode") or "" }}
+            {{ opencv_libs.append("xfeatures2d") or "" }}
+            {{ opencv_libs.append("ximgproc") or "" }}
+            {{ opencv_libs.append("xobjdetect") or "" }}
+            {{ opencv_libs.append("xphoto") or "" }}
+            {{ opencv_libs.append("freetype") or "" }}
+            - export MACOSX_DEPLOYMENT_TARGET={{ MACOSX_DEPLOYMENT_TARGET }}      # [osx]
+            - export CONDA_BUILD_SYSROOT={{ CONDA_BUILD_SYSROOT }}                # [osx]
+            - OPENCV_FLAGS=`pkg-config --cflags opencv4`                          # [unix]
+            - $CXX -std=c++11 test.cpp ${OPENCV_FLAGS} -o test        # [unix]
+            - if [[ $(./test) != $PKG_VERSION ]]; then exit 1 ; fi                # [unix]
+            {% for each_opencv_lib in opencv_libs %}
+            - echo Testing for presence of {{ each_opencv_lib }}
+            - test -f $PREFIX/lib/libopencv_{{ each_opencv_lib }}${SHLIB_EXT}     # [unix]
+            - if not exist %PREFIX%\\Library\\bin\\opencv_{{ each_opencv_lib }}{{ win_ver_lib }}.dll exit 1  # [win]
+            - echo Found                   {{ each_opencv_lib }}
+            {% endfor %}
+            - test -f $PREFIX/lib/libopencv_bioinspired${SHLIB_EXT}  # [unix]
+            - test -f $PREFIX/lib/libopencv_hdf${SHLIB_EXT}          # [unix]
+            - mkdir -p cmake_build_test && pushd cmake_build_test
+            - cmake -G "Ninja" ..
+            - cmake --build . --config Release
```

Comment:
worth it to actually run the built test binary?

Reply:
The only thing it does is print the version. It's really testing an executable can be built with opencv.

Reply:
fair